pkg/backend/httpstate/backend.go

```diff
@@ -209,8 +210,14 @@ func loginWithBrowser(ctx context.Context, d diag.Sink, cloudURL string, opts di
 
 	accessToken := <-c
 
+	username, err := client.NewClient(cloudURL, accessToken, d).GetPulumiAccountName(ctx)
+	if err != nil {
+		return nil, err
+	}
+
 	// Save the token and return the backend
-	if err = workspace.StoreAccessToken(cloudURL, accessToken, true); err != nil {
+	account := workspace.Account{AccessToken: accessToken, Username: username}
```

Comment:
Similar to other times when we create new accounts to store in the workspace, should we be setting `LastUsed` on the `AccountObject` here?

Reply:
Yes, we should--good catch.